client/src/view/components/Header/Nav/Logo/Logo.jsx

```diff
@@ -0,0 +1,28 @@
+import React, {useEffect, useState} from 'react';
+import { Link } from 'react-router-dom';
+import Box from '@material-ui/core/Box';
+import logo from '../icons/Originalité.svg';
+import logoDesktop from '../icons/OriginalitéDesktop.svg'
+import useWindowSize from "../../../../hooks/useWindowSize";
+import constants from "../../../../constants";
+
+function Logo() {
+    const [isDesktop, setIsDesktop] = useState()
+    const sizes = useWindowSize();
+
+    useEffect(() => {
+        sizes.width >= constants.WINDOW_DESKTOP_SIZE
```

Comment:
This can be simplified to`setIsDesktop(sizes.width >= constants.WINDOW_DESKTOP_SIZE )`